puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_hammer_plugin.yaml

```diff
@@ -0,0 +1,26 @@
+---
+- job-template:
+    name: test_hammer_plugin_{name}_{branch}
```

Comment:
To follow our conventions this should become:

{name}-pr-test

Example: https://ci.theforeman.org/job/katello-pr-test/